healthScore/healthScore/migrations/0002_appointment_user_alter_hospitals_contactinfo_and_more.py

```diff
@@ -0,0 +1,91 @@
+# Generated by Django 4.2 on 2024-02-20 21:58
+
+import datetime
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('healthScore', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='appointment',
+            fields=[
+                ('id', models.AutoField(primary_key=True, serialize=False)),
+                ('appointmentName', models.TextField()),
+                ('appointmentProperties', models.JSONField(null=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='user',
+            fields=[
+                ('id', models.AutoField(primary_key=True, serialize=False)),
+                ('email', models.EmailField(max_length=254)),
+                ('name', models.TextField()),
+                ('password', models.TextField()),
+                ('userName', models.TextField(unique=True)),
+                ('dob', models.DateField()),
+                ('contactInfo', models.TextField(default='', max_length=10)),
+                ('proofOfIdentity', models.TextField(null=True)),
+                ('address', models.TextField()),
+                ('securityQues', models.TextField(default='')),
+                ('securityAns', models.TextField(default='')),
+                ('gender', models.TextField(default='')),
+                ('profilePic', models.TextField(null=True)),
+                ('bloodGroup', models.TextField()),
+                ('requests', models.JSONField(null=True)),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='hospitals',
```

Comment:
Shouldn't model name be `hospital` here?

Reply:
Oh, this is auto generated. Once you make a chance in the models and run python manage.py migrate, it would generate it. 
This was before the changes that we had done. So the 0003 migration file should have the right information